contracts/connectors/loantoken/modules/LoanTokenSettingsLowerAdmin.sol

```diff
@@ -193,9 +195,14 @@ contract LoanTokenSettingsLowerAdmin is LoanTokenLogicStorage {
 					uint256(0xd46a704bc285dbd6ff5ad3863506260b1df02812f4f857c8cc852317a6ac64f2)
 				)
 			);
+		assembly {
+			paused := sload(slot)
+		}
+		require(paused != isPaused, "invalid");
```

Comment:
the require is not strictly required here, but doesn't hurt either. however, the error message doesn't provide any valuable information. change it to "isPaused is already set to that value" or similar message.

Reply:
Done